### PR TITLE
Feature/tests

### PR DESCRIPTION
**Pull request by MatHenriquez**
#  Summary
- Add ID tag attributes to LoginModal and InfoModal components to facilitate testing and make them readable.
- Create a component test to ensure the proper functioning of the Home component.

# Details
- Add testing for the Home page component.

# Evidences

## Home tests passing
home_tests

### CODE REVIEW

File: app/components/InfoModal.tsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ const InfoModal: FC<IKeyPair> = ({ publicKey, secretKey }: IKeyPair) => {
           <div className="flex flex-shrink-0 items-center justify-between rounded-t-md border-b-2 border-neutral-100 border-opacity-100 p-4 dark:border-opacity-50">
             <h5
               className="text-xl font-medium leading-normal text-neutral-800 dark:text-neutral-200"
-              id="exampleModalCenterTitle"
+              id="ModalTitle"
```
Comment (Genebson) [inline app/components/InfoModal.tsx:25]: Id's must be written in `kebab-case`

**Review by Genebson — APPROVED** [review]

File: app/page.cy.tsx [hunk 2]
```diff
@@ -19,4 +19,23 @@ describe("Home", () => {
       "Create new keys and Sign Up"
     );
   });
+
+  it("should show the keys when clicking the create new keys link", () => {
```
Comment (Genebson) [inline app/page.cy.tsx]: This and every `it` has to be capitalized.

File: app/components/InfoModal.tsx (hunk 1 above)
Comment (MatHenriquez) [reply app/components/InfoModal.tsx:25]: I have changed the IDs to be in kebab-case

File: app/page.cy.tsx (hunk 2 above)
Comment (MatHenriquez) [reply app/page.cy.tsx]: I have capitalized the descriptions of the tests.

**Pull request closed without merging**